packages/vip-go/services/logService.js

```diff
@@ -56,7 +54,7 @@ const getService = (namespace) => {
     sentryConfig = require('@irvingjs/config/sentryConfig');
   } else {
     const getSentryConfig = require('../config/getSentryConfig');
-    sentryConfig = getSentryConfig();
+    sentryConfig = require(getSentryConfig());
```

Comment:
Could maybe rename this function to `getSentryConfigPath` as well, for your future self.